conf/init/ol-solr-updater.conf

```diff
@@ -6,5 +6,6 @@ stop on runlevel [!2345]
 
 chdir /openlibrary
 respawn
```

Comment:
@mekarpeles how much do you want to respawn the default is set as 
 Default COUNT is 10. Default INTERVAL is 5 seconds. 
If you want to respawn many times use -
respawn limit unlimited